.env.example

```diff
@@ -0,0 +1,14 @@
+# Django secret key used on app settings
+SECRET_KEY="your_secret_here"
+
+# authentication for koiki API.
+KOIKI_HOST="https://rekistest.koiki.es/services"
+KOIKI_AUTH_TOKEN="your_auth_token here"
+
+# wordpress woocommerce auth
+WCFMMP_HOST="your_wp_woocomerce_url"
+WCFMMP_USER="your_wp_woocomerce_user"
+WCFMMP_PASSWORD="your_wp_woocomerce_password"
+# wordpress woocomerce secret used on webhooks
```

Comment:
This is just my OCD, but I would add a space between these two blocks, so above this comment.

Reply:
Again, to be a bit more specific, what about `Details to authenticate Woocommerce's incoming webhooks`.

Reply:
Fixed by: https://github.com/coopdevs/lazona_connector/pull/92/commits/2881bc3cc316a5113ff727b6609fb44e3b41c9e8